CommonTools/Utils/src/FormulaEvaluator.cc

```diff
@@ -581,6 +581,32 @@ namespace {
   const std::string k_TMath__Sqrt("TMath::Sqrt");
   const std::string k_abs("abs");
   const std::string k_TMath__Abs("TMath::Abs");
+  const std::string k_cos("cos");
+  const std::string k_sin("sin");
```

Comment:
please group TMath and "std" variants in the same way they appear already for other functions.

Reply:
hi @slava77 , i have modified those codes as per your suggestion.